### DISCUSSION TITLE
Skip package and dependencies on a specific platform

### DISCUSSION THREAD

**Discussion by jbcpollak** [vercel/turborepo#8770] [Help]
### Summary

I'd like to skip an entire package and any part of its dependency tree (if not needed by other packages) based on platform. For example, I'd like to avoid building something on MacOS but always build it on Linux.

I can use `turbo build --filter=\!app` on the command line, but is there a way to set the filter in an environment variable or another way I can skip/disable the project programmatically so I can automatically disable it based on certain parameters?

### Example

```
turbo build -F=\!./disabled-app
```

(Side note: I seem to have to escape the !, but I noticed other users don't need to, I'm not sure why that is.)

**Comment by anthonyshew** [comment]
This isn't a feature that we're likely to focus on, but you can create tasks with names like `build:linux` or `build:macos`, if you'd like to partition them.

Escaping `/` is needed depending on the terminal you're using.
